### PR TITLE
Turn on alert on click

### PR DESCRIPTION
**Pull request by mgerring**
The branch name is a misnomer, it doesn't toggle. If alerts are turned off, alerts are turned on. If they're already on, nothing happens.

### CODE REVIEW

File: assets/js/AddNotificationButtonClosed.js [hunk 1]
```diff
@@ -45,6 +47,20 @@ class AddNotificationButtonClosed extends React.Component {
     this.setState({showButton: false});
   }
 
+  handleClick() {
```
Comment (jstray) [inline assets/js/AddNotificationButtonClosed.js]: I'd already forgotten what this logic does and had to read it to figure out -- would be useful to have a short comment like "Add notification to the first module that adds data."

File: assets/js/AddNotificationButtonOpen.js [hunk 2]
```diff
@@ -6,8 +6,9 @@
  */
 
 import React from 'react';
-import PropTypes from 'prop-types'
+import PropTypes from 'prop-types' //TODO: actually add prop types
```
Comment (jstray) [inline assets/js/AddNotificationButtonOpen.js]: Why not add them now? They are useful for detecting bugs.

File: assets/js/AddNotificationButtonOpen.test.js [hunk 3]
```diff
@@ -10,17 +10,79 @@ import AddNotificationButtonOpen  from './AddNotificationButtonOpen'
...
+  var wrapper;
+  var notificationsOn;
+  var notificationsOff;
 
-  beforeEach(() => wrapper = mount(
-    <DragDropContextProvider backend={HTML5Backend}>
-      <AddNotificationButtonOpen/>
-    </DragDropContextProvider>
-  ));
+  beforeEach(() => {
```
Comment (jstray) [inline assets/js/AddNotificationButtonOpen.test.js]: This looks nearly identical to the closed test. Can it be factored out somewhere? Is it generic enough to become a standard test workflow in utils? If not even just sharing between these two tests is a good idea. Cut and paste code gives me hives.

**Comment by mgerring** [comment]
Boy, that was an ugly merge

**Comment by mgerring** [comment]
Manually merged

**Pull request closed without merging**